ti/savings/cache/gradle/parser.go

```diff
@@ -0,0 +1,249 @@
+package gradle
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/harness/ti-client/types"
+	"github.com/mattn/go-zglob"
+	"github.com/sirupsen/logrus"
+	"golang.org/x/net/html"
+)
+
+func ParseSavings(workspace string, log *logrus.Logger, logErr bool) (types.IntelligenceExecutionState, int, error) {
+	cacheState := types.FULL_RUN
+	totalBuildTime := 0
+
+	path := fmt.Sprintf("%s/build/reports/profile/*.html", workspace)
+	files, err := zglob.Glob(path)
+	if err != nil {
+		if errors.Is(err, os.ErrNotExist) {
+			return cacheState, totalBuildTime, fmt.Errorf("no profiles present")
+		}
+		if logErr {
```

Comment:
This will only be hit if its not a file exists error and we want to log. Why do we want to selectively log? When will it be used?

Reply:
Removed `logErr` boolean. If profile is not present, we'll try to look for other build caches. If present and there's some issue with the path, we need to log it.